app/views/change_requests/index.html.erb

```diff
@@ -0,0 +1,84 @@
+<% content_for :title, "Requests for changes" %>
+<% add_parent_breadcrumb_link "Home", planning_applications_path %>
+<% add_parent_breadcrumb_link "Application", validate_documents_form_planning_application_path(@planning_application) %>
+
+<div class="govuk-grid-column-full">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column govuk-!-padding-top-6">
+      <fieldset class="govuk-fieldset">
+        <h2 class="govuk-heading-l">
+          Requests for changes
+        </h2>
+        <% if @planning_application.change_requests %>
```

Comment:
Should be `@planning_application.change_requests.present?`, as will return truthy with an empty array otherwise.